### PR TITLE
Birthday

### PR DESCRIPTION
**Pull request by kvigen**
Swagger API definitions for APIs v1.1 and v1.2

### CODE REVIEW

**Comment by kvigen** [comment]
cc @drhurd @arsalansufi @meliaj @ameliez

File: v1.1.yml [hunk 1]
```diff
@@ -0,0 +1,1572 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SectionResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Get only a specific section's information.
+
```
Comment (beenakamath) [inline v1.1.yml]: minor nit: remove empty line for uniformity with rest of the code.

File: v1.1.yml [hunk 2]
```diff
@@ -0,0 +1,1572 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/DistrictResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Retrieves district information for a section.
+
```
Comment (beenakamath) [inline v1.1.yml]: minor nit: remove empty line for uniformity with rest of the code.

File: v1.1.yml [hunk 3]
```diff
@@ -0,0 +1,1572 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/DistrictResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Retrieves district information for a student.
+
```
Comment (beenakamath) [inline v1.1.yml]: minor nit: remove empty line for uniformity with rest of the code.

File: v1.1.yml [hunk 4]
```diff
@@ -0,0 +1,1572 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/DistrictResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Retrieves district information for a teacher.
+
```
Comment (beenakamath) [inline v1.1.yml]: minor nit: remove empty line for uniformity with rest of the code.

File: v1.1.yml [hunk 5]
```diff
@@ -0,0 +1,1572 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/GradeLevelsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Retrieves a list of all grade levels taught by a specific teacher.
+
```
Comment (beenakamath) [inline v1.1.yml]: minor nit: remove empty line for uniformity with rest of the code.

File: v1.1.yml [hunk 6]
```diff
@@ -0,0 +1,1572 @@
...
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SchoolAdminResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Get only a specific schooladmin's information.
+
```
Comment (beenakamath) [inline v1.1.yml]: minor style nit: remove empty line for uniformity with rest of the file.

File: v1.1.yml [hunk 7]
```diff
@@ -0,0 +1,1572 @@
...
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/DistrictAdminResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+      summary: Retrieves a district admin
+  /districts:
+    get:
```
Comment (beenakamath) [inline v1.1.yml]: why doesn't getDistricts have starting_after, end_before etc?

Comment (kvigen) [inline v1.1.yml]: good catch. it should

**Comment by kvigen** [comment]
i also prefer an empty line. will standardize on that

**Pull request merged**